### PR TITLE
Update README for getting cf_clearance token and user agent

### PR DESCRIPTION
**Pull request by kx-Huang**
Add the README for getting cf_clearance token and user agent, as well as their configurations in Railway deployment.

### CODE REVIEW

File: src/chatgpt.ts [hunk 1]
```diff
@@ -302,15 +302,15 @@ export class ChatGPTBot {
     }
   }
   // Check whether the ChatGPT processing can be triggered
-  tiggerGPTMessage(text: string, privateChat: boolean = false): boolean {
-    const chatPrivateTiggerKeyword = this.chatPrivateTiggerKeyword;
+  triggerGPTMessage(text: string, privateChat: boolean = false): boolean {
+    const chatPrivateTriggerKeyword = this.chatPrivateTriggerKeyword;
```
Comment (fuergaosi233) [inline src/chatgpt.ts:306]: Should this be done in another PR?

Comment (kx-Huang) [reply src/chatgpt.ts]: Noted. Revert to last commit
